Std/Tactic/Omega/IntList.lean

```diff
@@ -35,9 +35,9 @@ theorem get_of_length_le {xs : IntList} (h : xs.length ≤ i) : xs.get i = 0 :=
   rw [get, List.get?_eq_none.mpr h]
   rfl
 
-theorem lt_length_of_get_nonzero {xs : IntList} (h : xs.get i ≠ 0) : i < xs.length := by
-  revert h
-  simpa using mt get_of_length_le
+-- theorem lt_length_of_get_nonzero {xs : IntList} (h : xs.get i ≠ 0) : i < xs.length := by
+--   revert h
+--   simpa using mt get_of_length_le
```

Comment:
What is the intent with the commented out code?  Could it be deleted?

Reply:
Yes: last para of the PR description. I found lots of proofs in `Tactic/Omega/` are just not used. I commented them out so they didn't cause spurious dependencies, but I didn't want to take the time right now to decide if anything is worth saving.

Reply:
But I think I'll just delete it.